src/pages/DashboardPage.jsx

```diff
@@ -1,13 +1,38 @@
-import AddEventBanner from "../components/AddEventBanner";
-import EventsList from "../components/EventsList";
-
+import addTeam from "../assets/add-team.svg";
+import share from "../assets/share-08.svg";
+import Button from "../components/Button";
+import EventCard from "../components/EventCard";
+import Footer from "../components/Footer";
+import HostsCard from "../components/HostsCard";
+import VisibilityCard from "../components/VisibilityCard";
 function DashboardPage() {
```

Comment:
link this page to a route

Reply:
"/home" is not throwing blank page,

Reply:
Current route is "/app/home"
changing to /home would affect sidebar and layout settings